src/data/markdown/translated-guides/en/02 Using k6/02 Metrics/100 Create custom metrics.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Create custom metrics
+description: How to build custom k6 metrics for each metric type.
+---
+
+Besides the [built-in metrics](/results-output/metrics-reference), you can create custom metrics.
+For example, you can create some derived metrics from the built-in metrics, or use the [Response.timings](/javascript-api/k6-http/response) object to create a metric for a specific set of endpoints.
```

Comment:
No it'll be joined in one paragraph. One sentence per line is pretty standard in technical docs to keep diffs confined to sentences instead of paragraphs: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line